mods/seasonal/moves.js

```diff
@@ -166,8 +166,8 @@ let BattleMovedex = {
 		accuracy: 100,
 		basePower: 70,
 		category: "Physical",
-		desc: "",
-		shortDesc: "",
+		desc: "If the target uses certain non-damaging moves this turn, the user steals that move to use itself. Fails if no move is stolen or if the user is under the effect of Sky Drop.",
+		shortDesc: "Steals status move; fails if it does not. Priority.",
```

Comment:
Fails if target is attacking. Steals foes move. Priority.

Reply:
Steals foe's move. Fails if target attacks. Priority.
(order for clarity, phrasing to extra-make clear that they're using a damaging move as opposed to merely acting)